src/styles/helpers/reset.scss

```diff
@@ -0,0 +1,21 @@
+body {
+  margin: 0;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+ul {
+  padding: 0;
+  margin: 0;
+}
+
+h2 {
+  margin: 0;
+}
```

Comment:
Identical styles can be specified by writing tags/classes separated by commas to avoid duplication of code.